crates/bevy_audio/src/audio.rs

```diff
@@ -141,30 +151,40 @@ impl PlaybackSettings {
         self.speed = speed;
         self
     }
+
+    /// Helper to enable or disable spatial audio.
+    pub const fn with_spatial(mut self, spatial: bool) -> Self {
+        self.spatial = spatial;
+        self
+    }
 }
 
-/// Settings for playing spatial audio.
+/// Settings for the listener for a spatial audio source.
 ///
-/// Note: Bevy does not currently support HRTF or any other high-quality 3D sound rendering
-/// features. Spatial audio is implemented via simple left-right stereo panning.
+
```

Comment:
should delete this empty line and the one above it.

Reply:
Those docs on this and `new` were a bit funky with leftover bits from the old implementation. Should be making more sense now.